plugins/devtools-common/src/types.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import { JsonValue } from '@backstage/types';
+import { JSONObject } from '@apollo/explorer/src/helpers/types';
```

Comment:
Could we not use the `JsonObject` from `@backstage/types`?

Reply:
Oh yes, good catch!